main.go

```diff
@@ -124,13 +131,32 @@ func enableOnCI() {
 	}
 }
 
-func setPalette() {
+func parseEnvAndSetPalette() {
+	v := os.Getenv(paletteEnv)
+	if v == "" {
+		parseEnvColor()
+	} else {
```

Comment:
Using if / else here prevents us from using both sets of env variables.

For example, if `GOTEST_PALETTE` is set to custom colors in our default environment, but we'd like to change the color of skipped tests for this run only, we get some unexpected behavior: The skipped test output will be colored as expected, but the passing and failed tests will be set to default colors (not the custom ones defined in our environment)

Reply:
I understand what you mean, but how would you handle precedence, both color definitions are based on environment variable definitions, meaning we cannot resolve the origin or order.

As I see it a simple rule dictating order must be implemented and documented.

I will do some experimentation.

Reply:
I would give precedence to the color specific variables over `GOTEST_PALETTE` because their functionality is more narrow. I agree about documentation and think defining this behavior in the readme should be sufficient.

Reply:
hi @subtlepseudonym 

Thanks for the swift response, I agree. I will give it a shot code first, the documentation.